tests/rptest/tests/kafka_client_compat_test.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark import matrix
+from rptest.tests.redpanda_test import RedpandaTest
+
+from rptest.clients.types import TopicSpec
+from rptest.clients.kafka_cli_tools import KafkaCliTools
+
+
+class KafkaClientCompatTest(RedpandaTest):
+    def test_create_topic(self):
+        for client_factory in KafkaCliTools.instances():
+            client = client_factory(self.redpanda)
+            topics = [TopicSpec() for _ in range(3)]
```

Comment:
this line is confusing to me. you are creating 3 default topics. is the name auto generated?

Reply:
yes